web-dapp/server-lib/sign.js

```diff
@@ -48,7 +48,7 @@ function sign3(text) {
 
 var secp256k1 = require('secp256k1');
 
-function sign(text) {
+function sign(text, privateKey) {
     logger.log(prelog + 'signer: ' + config.signer);
```

Comment:
This line will be misleading if `privateKey` does not actually belong to `config.signer`. Printing the whole key is probably not a good idea, on the other hand, it would be nice to have some indication in logs who's signing the message. Maybe print first few (6?) symbols of the private key, what do you think?

Reply:
Honestly, I would just remove that log. But if you think being able to identify the private key from the logs may be useful, logging the first 4 characters sounds fine. It makes an attack somewhat easier if someone gets access to the logs, though, but that may be an acceptable trade-off, I don't know.

Reply:
I agree, let's remove it

Reply:
Removed.